### ISSUE TITLE
Marta the Mysterious Movement Muncher: Single-key bindings for core.move.up/down are eaten

### ISSUE THREAD

**Issue by grossjohann** [marta-file-manager/marta-issues#671]
I have `behavior/quickSelect/activateOnAnyCharacter = false`, and I have the following keybindings (excerpt):

```
keyBindings {
    "J" { id "core.move.down" }
    "K" { id "core.move.up" }
    "L" { id "core.open" }
    "H" { id "core.go.up" }
    "E" { id "core.edit" }
    "." { id "core.actions" }
}
```

So I type `/` to enable quick select mode, and now characters I type show up in the status bar (and it also searches for the corresponding file name, of course).  If I type `/model`, then I see the string "model" in the status bar.  But if I type `/jar`, then I see "ar" -- the "j" was eaten!  It turns out that `j` and `k` are eaten like this, but `l`, `h`, `e`, `.` are not eaten.

Could you give me my J and K back in Quick Select?

**Comment by yanex** [comment]
Hello @grossjohann!

Thanks for pointing out this inconsistency!
`J` and `K` hotkeys work as they are mapped to base actions that have the highest priority. Since 0.8, base actions won't have such a privilege, so hotkeys will work as they work with `L`, `H` and `E`.

However, it doesn't mean there won't be a way to achieve what you want. In fact, it's possible even with the current version of Marta. There's a configuration key that disables Quick Select activation on letter keys:

```
behavior {
    quickSelect {
        activateOnAnyCharacter false
    }
}
```

When `activateOnAnyCharacter` is `false`, you will need to press `/` (and then type something) to activate Quick Select.

**Comment by kgrossjo** [comment]
I'm happy to wait for 0.8, but I would like to point out that I've already set activateOnAnyCharacter to false.  And the behavior is as expected: I have to hit `/` to enter quick select mode.

**Comment by yanex** [comment]
Oh sorry! You're right.
Still, the bug will be fixed in 0.8.

**Comment by kgrossjo** [comment]
Thanks a lot for all your work, I'm looking forward to 0.8!
